package.json

```diff
@@ -5,9 +5,9 @@
     "cron": "^2.1.0",
     "env-var": "^7.3.0",
     "express-http-context": "^1.2.4",
+    "@greyoda/ldes-client": "0.0.15",
```

Comment:
Currently it's using my fork which lives here: https://github.com/sergiofenoll/ldes-client

At the moment of writing, the fork implements `loose` mode for extracting members. This ensures that certain misconfigured LDES feeds can still be parsed.

Other issues with the client are:

- Versioning doesn't work when the state is retrieved from the serialized state file (see: https://matrix.to/#/!HCYaBzWmkpZlPcCyAL:chat.semantic.works/$5uy83w1PnVNWzRNQ4O3CWs5SyY2mmh6u6qwP4GwnHg4?via=chat.semantic.works&via=matrix.org&via=gitter.im)
- `ldes:versionOfPath` and `ldes:timestampPath` aren't exposed to consumers